pkg/genericcli/prompt.go

```diff
@@ -53,11 +69,45 @@ func PromptCustom(c *PromptConfig) error {
 		c.Out = os.Stdout
 	}
 
+	// validate, we need to panic here because this is really a configuration error and code execution needs to stop
+	for _, answer := range c.AcceptedAnswers {
+		if len(answer) == 0 {
+			panic("configured prompt answer must not be an empty string")
+		}
+	}
+
+	defaultAnswerIndex := slices.IndexFunc(c.AcceptedAnswers, func(answer string) bool {
+		return answer == c.DefaultAnswer
+	})
+
+	if c.DefaultAnswer != "" {
+		if defaultAnswerIndex < 0 && c.DefaultAnswer != c.No {
+			panic("configured prompt default answer must be contained in accepted answer or no answer")
+		}
+	}
+
 	if c.ShowAnswers {
+		firstCharToUpper := func(s string) string {
```

Comment:
Please use a caser for this: https://pkg.go.dev/golang.org/x/text@v0.19.0/cases#Title

Reply:
Caser is language-specific. I do not want this to be specific to any language.

Reply:
> Caser is language-specific. I do not want this to be specific to any language.

But cloudctl is english only, so no Prblem

Reply:
I don't know, for me this is too opinionated. It also transforms other letters of the word like "yeS" it will turn to "Yes". It's somehow not want I want. :D

Reply:
Then a rename would be fine for me

Reply:
x/text is not considered as a experimental library, it is kept outside the standard library on purpose to get fixes and new features faster

Reply:
> Then a rename would be fine for me

Renamed to sentence case: https://en.wikipedia.org/wiki/Letter_case#Sentence_case